activation/atxdb.go

```diff
@@ -524,11 +524,12 @@ func (db *DB) storeAtxUnlocked(atx *types.ActivationTx) error {
 		return err
 	}
 
-	atxBodyBytes, err := types.InterfaceToBytes(getAtxBody(atx))
+	// todo: this changed so that a full atx will be written - inherently there will be double data with atx header
```

Comment:
@antonlerner what's the advantage of storing the header fields twice? one instead of two db hits when syncing?